### ISSUE TITLE
Issue with include evaluation of filename if namespace is used

### ISSUE THREAD

**Issue by Doomerdinger** [ros/xacro#263]
**example.xacro**
```xacro
<?xml version="1.0" ?>
<robot name="example" xmlns:xacro="http://www.ros.org/wiki/xacro">
  <xacro:property name="include_file" value="$(find example_package)/child.xacro"/>

  <xacro:include filename="${include_file}" ns="child_ns"/>
</robot>
```
The above xacro file will fail, complaining that `include_file` is not defined. This error does not present itself if the `ns` attribute is not used.

**example2.xacro**
```xacro
<?xml version="1.0" ?>
<robot name="example" xmlns:xacro="http://www.ros.org/wiki/xacro">
  <xacro:arg name="include_file" default="$(find example_package)/child.xacro"/>

  <xacro:include filename="$(arg include_file)" ns="child_ns"/>
</robot>
```
The above file appears to work just fine.

**Comment by rhaschke** [comment]
Thanks for reporting this issue. I filed a fix in #264.

**Comment by rhaschke** [comment]
Closed via #264.
